server/container/guice/guice-common/src/main/java/org/apache/james/jmap/JMAPModule.java

```diff
@@ -106,6 +106,8 @@ public void initModule() {
                     "MOVE support in MailboxManager is required by JMAP Module");
             Preconditions.checkArgument(mailboxManager.getSupportedMessageCapabilities().contains(MailboxManager.MessageCapabilities.Attachment),
                     "Attachment support in MailboxManager is required by JMAP Module");
+            Preconditions.checkArgument(mailboxManager.getSupportedSearchCapabilities().contains(MailboxManager.SearchCapabilities.MultimailboxSearch),
```

Comment:
I don't think, as:
- MessageSearchIndex is not really linked to MailboxManager, not as MessageManager
- MessageSearchIndex as it's own configuration file `indexer.xml`
- MessageSearchIndex implementation is not coupled to MailboxManager implementation, so you can have James running on JPA with ES